Signal/src/call/CallService.swift

```diff
@@ -1205,6 +1202,10 @@ protocol CallServiceObserver: class {
     public func handleFailedCall(failedCall: SignalCall?, error: CallError) {
         AssertIsOnMainThread()
 
+        if case .assertionError(let description) = error {
+            assertionFailure(description)
```

Comment:
+1

I'd like to add a swift free function analogous to OWSFail() that `assertionFailure` in DEBUG and DDLogError() in production.